src/Umbraco.Cms.Integrations.Automation.Zapier/App_Plugins/UmbracoCms.Integrations/Automation/Zapier/dashboard.html

```diff
@@ -22,50 +23,33 @@
                     You can initiate your automation when a content item of a particular document type is published in Umbraco.
                 </p>
                 <p>
-                    Using the filters below, map content items with Zap triggered webhooks.  This will:
+                    The integration uses Zapier subscription hook triggers, allowing Zapier to set up and remove hook subscriptions through Subscribe/Unsubscribe API requests.
```

Comment:
```suggestion
                    The integration uses Zapier subscription hook triggers, allowing Zapier to set up and remove hook subscriptions when Zaps are created or removed on the platform.
```

Just a little less technical.